conflation/aggregation.py

```diff
@@ -0,0 +1,139 @@
+import copy
+import json
+import os
+import pandas as pd
+import pickle
+
+from mapillary import util
+
+MAP_MATCH_COLS = ["density", "road_class", "type", "kph"]
+# Basic config from OpenStreetMapSpeeds/schema repo where all values are None.
+BASE_CONFIG = {
+    "rural": {
+        "way": [None, None, None, None, None, None, None, None],
+        "link_exiting": [None, None, None, None, None],
+        "link_turning": [None, None, None, None, None],
+        "roundabout": [None, None, None, None, None, None, None, None],
+        "driveway": None,
+        "alley": None,
+        "parking_aisle": None,
+        "drive-through": None,
+    },
+    "suburban": {
+        "way": [None, None, None, None, None, None, None, None],
+        "link_exiting": [None, None, None, None, None],
+        "link_turning": [None, None, None, None, None],
+        "roundabout": [None, None, None, None, None, None, None, None],
+        "driveway": None,
+        "alley": None,
+        "parking_aisle": None,
+        "drive-through": None,
+    },
+    "urban": {
+        "way": [None, None, None, None, None, None, None, None],
+        "link_exiting": [None, None, None, None, None],
+        "link_turning": [None, None, None, None, None],
+        "roundabout": [None, None, None, None, None, None, None, None],
+        "driveway": None,
+        "alley": None,
+        "parking_aisle": None,
+        "drive-through": None,
+    },
+}
+ROAD_CLASS_INDEX_MAPPING = {
+    "motorway": 0,
+    "trunk": 1,
+    "primary": 2,
+    "secondary": 3,
+    "tertiary": 4,
+    "unclassified": 5,
+    "residential": 6,
+    "service_other": 7,
```

Comment:
we should make it so that the name coming out of the map matching phase is `service` rather than `service_other` as the latter is valhalla-specific